dev_docs/developer_scripts.md

```diff
@@ -118,3 +70,65 @@ gui-spec-windows - creates the initial spec file for windows, do not run unless
 gui-build-osx - builds the app from the osx spec file
 gui-build-windows - builds the app from the windows spec file
 ```
+## Client Makefile
+
+The following phony `make` targets in `client/Makefile` are convenience methods for getting you up and developing.
+
+### Runner scripts
+
+#### start-frontend
+
+**About** Serve the current client javascript independently from the `server` code.
+
+**Requires**
+* The server to be running. Best way to do this is with `make backend-dev`
+* `make ci` to install the necessary node modules
+
+**Usage** `make start-frontend`
+
+#### backend-dev
+
+**About** This script enables FE developers to run the REST API necessary to back the development server for the front end. It is intended to ensure that the FE developer gets the current version of the backend with a single command and no knowledge of python necessary. It creates and activates a virtual environment and installs cellxgene from the current branch.
```

Comment:
suggest documenting the use of CXG_OPTIONS and provide an example.  Without this, we are going to have a explosion of additional targets like `backend-dev-anno`, `backend-dev-anno-ontology`, etc.

Reply:
True. Added.